src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -1257,6 +1257,101 @@ void Snapshot::repeat(detail::Pass&& pass) {
     LOG_INFO("Number of groups after compiler pass: " << graphSize());
 }
 
+bool Snapshot::isRegularResultCase() const {
```

Comment:
@dmatveev, tbh, I'm still not sure this is the right place. Please confirm whether this is sufficient, or whether I should convert this to a free function and move it, for example, to `src/plugins/intel_npu/src/plugin/npuw/partitioning/online/compiler.cpp`